packages/block-library/src/latest-posts/index.php

```diff
@@ -151,6 +143,24 @@ function render_block_core_latest_posts( $attributes ) {
 
 			$trimmed_excerpt = get_the_excerpt( $post );
 
+			/*
+			 * Adds a "Read more" link with screen reader text.
+			 * [&hellip;] is the default excerpt ending from wp_trim_excerpt() in Core.
+			 */
+			if ( str_ends_with( $trimmed_excerpt, ' [&hellip;]' ) ) {
+				$excerpt_length = (int) apply_filters( 'excerpt_length', $block_core_latest_posts_excerpt_length );
+				if ( $excerpt_length <= $block_core_latest_posts_excerpt_length ) {
+					$trimmed_excerpt  = substr( $trimmed_excerpt, 0, -11 );
+					$trimmed_excerpt .= sprintf(
```

Comment:
@andrewserong Core generally trusts its translations so we tend not to run them through escaping functions. Since the translated string uses the default domain rather than a `'gutenberg'` domain, it should likely be treated the same as any other Core translation.

Reply:
Excellent, thanks for confirming @costdev! 🙂

I'll click "resolve" on this thread.